woocommerce/woocommerce-admin
# Add filter by variations in reports note

Fixes #5116

~Note that this is still missing the documentation URL for the new feature however the rest is fine for review, I'll just replace the URL once it turns up.~ This is ready for review.

This adds a note about the variations filter in the orders and products reports, which is shown for stores with at least one variable product.

### Screenshots
![image](https://user-images.githubusercontent.com/224531/94648833-ba152f80-0336-11eb-87b0-1b832e7bdc7c.png)

### Detailed test instructions:

- Don't have any variable products. You can do this by Products -> Filter by product type -> Variable products, then deleting all the products in bulk.
- Run the `wc_admin_daily` cron task
- The "filter by product variations in orders and products reports" note should _not_ appear
- Add a variable product (or restore an existing deleted variable product)
- Run the `wc_admin_daily` cron task
- The "filter by product variations in orders and products reports" note should now appear.

### Changelog Note:

Dev: Add filter by variations in reports note
